### PR TITLE
Auto-release PDFs

### LINKED ISSUE #6: References missing in PDF on GitHub
Who and when will update the PDF? How do we avoid probelems like this in the future?

Maybe having the PDF as a release rather than inside the repo would be better? Or we could consider some webservice that automatically updates the PDF on each commit.

### PR DESCRIPTION
**Pull request by anishathalye**
Fixes #6.

This PR adds Travis CI to automatically build and release each commit on the master branch.

Another minor change, we include the git version hash in the PDF generated.

### CODE REVIEW

**Comment by anishathalye** [comment]
(not sure why the CI says one of the two builds was not successful; I think it should be okay to merge)

**Comment by carlini** [comment]
I'm somewhat concerned about the build failing. Do you know why it's failing at least?

**Comment by anishathalye** [comment]
Yes -- I've seen something like this before -- sometimes Travis CI gets confused when you force push to a branch. If you take a look at the [build log](https://travis-ci.org/evaluating-adversarial-robustness/adv-eval-paper/builds/496303552), it errors when trying to check out the pull request commit (and I can't get it to re-run the build and succeed) -- the error doesn't really have anything to do with our CI settings in particular.

I just made the commit hash change (`git commit --amend && git push -f`, with no real changes), and now the build is [working](https://travis-ci.org/evaluating-adversarial-robustness/adv-eval-paper/builds/496318997).

File: paper.tex [hunk 1]
```diff
@@ -57,7 +61,7 @@
 Please direct correspondence to the GitHub repository \\
 \url{https://github.com/evaluating-adversarial-robustness/adv-eval-paper} \\
 \noindent \\
-Last Update: 18 February, 2019. \\ %TODO CHANGE OTHER DATE TOO
+Last Update: 18 February, 2019 (revision \texttt{\version}).
```
Comment (jonasrauber) [inline paper.tex]: @anishathalye great work! Shouldn't the date also be updated automatically?

Comment (anishathalye) [reply paper.tex]: That's a good point that we should discuss. I wasn't sure if this should be <date paper was compiled>, or <date last substantial change was made to the paper>. Perhaps doing this automatically would be better. What do others think?

**Comment by carlini** [comment]
It depends if we want this date to increase every time anything at all changes, always, or if we want to not change it if, e.g., we fix a typo and recompile the pdf.

**Comment by jonasrauber** [comment]
What's the harm of updating the date when we update the revision number anyway for each change?

**Comment by anishathalye** [comment]
I'm in favor of updating the date automatically along with the revision number. I'll push a commit to this PR that makes that change.

**Comment by carlini** [comment]
We should make the change here too then

https://github.com/evaluating-adversarial-robustness/adv-eval-paper/blob/master/paper.tex#L710

**Comment by anishathalye** [comment]
Good catch; fixed.

**Comment by anishathalye** [comment]
Does this look okay to merge now?

**Comment by anishathalye** [comment]
Not sure why the release part is not working in this org. Seems to work in my local repo ([build](https://travis-ci.org/anishathalye/adv-eval-paper/builds/496689024) [release](https://github.com/anishathalye/adv-eval-paper/releases)). I'll try to track down the bug.

**Comment by anishathalye** [comment]
Bot needed write permissions to create a new release (so the deploy step was failing); fixed now (first deploy [here](https://github.com/evaluating-adversarial-robustness/adv-eval-paper/releases/tag/20190221202001-d487f66))

**Pull request merged**